### PR TITLE
Parallelize 4: Add fixes to the malted directory

### PR DESCRIPTION
**Pull request by DarshalShetty**

### CODE REVIEW

**Comment by themetaschemer** [comment]
This seems to include commits from the previous PR as well? Probably need to recreate after merging?

**Comment by DarshalShetty** [comment]
Rebase done

File: malted/test/test-E-gd-common.rkt [hunk 1]
```diff
@@ -1,12 +1,13 @@
 (module+ test
   (require rackunit)
+  (require "../impl.rkt")
 
-  (check-equal? (zeroes (tensor 1 2 3))
+  (check-dual-equal? (zeroes (tensor 1 2 3))
                 (tensor 0.0 0.0 0.0))
```
Comment (themetaschemer) [inline malted/test/test-E-gd-common.rkt:6]: Indentation broke here?

**Review by themetaschemer — APPROVED** [review]
Looks good. Please fix the one indentation issue.

**Pull request merged**